homeassistant/components/ness_alarm/__init__.py

```diff
@@ -109,6 +113,14 @@ async def _close(event):
 
     hass.bus.async_listen_once(EVENT_HOMEASSISTANT_STOP, _close)
 
+    async def _started(event):
+        # Force update for current arming status and current zone states (once Home Assistant has finished loading required sensors and panel)
+        _LOGGER.debug("%s event - invoking client keepalive() & update()", EVENT_HOMEASSISTANT_STARTED)
+        hass.loop.create_task(client.keepalive())
+        hass.loop.create_task(client.update())
+
+    hass.bus.async_listen_once(EVENT_HOMEASSISTANT_STARTED, _started)                              
```

Comment:
Don't do this, use either the `homeassistant.start.async_at_start` or `homeassistant.start.async_at_started` helper instead.